paracord-ui/src/Components/NavigationBar.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FunctionComponent } from 'react'
+import { Navbar, NavbarBrand } from 'reactstrap'
+
+const NavigationBar: FunctionComponent<{}> = () => {
+  return (
+    <Navbar color="dark" dark expand="md">
```

Comment:
Do we need 'color="dark"' and "dark"?

Reply:
Unfortunately yes, the `color="dark"` makes the background show up and then the plain `dark` applies styles to things inside the NavBar like text to make it look nice on the dark background. Not sure why they made it that way 🤷 lol